### DISCUSSION TITLE
Welcome to scikit-build Discussions!

### DISCUSSION THREAD

**Discussion by henryiii** [scikit-build/scikit-build#600] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hangarr** [comment]
I'm investigating _skbuild_ to build Python C++ extensions that use _opencv_.   We currently use _setuptools_ and _pybind11_, so other developers here are somewhat comfortable with that, but we are not expert as an organization doing this by any means.  It looks like _skbuild_ could be useful to us from these very simple examples:

> [**scikit-build-sample-projects**](https://github.com/scikit-build/scikit-build-sample-projects)
> `[](https://github.com/scikit-build/scikit-build-sample-projects)`
> 
> [**core-pybind11-hello**](https://github.com/scikit-build/scikit-build-sample-projects/tree/master/projects/core-pybind11-hello)
> `[](https://github.com/scikit-build/scikit-build-sample-projects/tree/master/projects/core-pybind11-hello)`
> 
> [**hello-pybind11**](https://github.com/scikit-build/scikit-build-sample-projects/tree/master/projects/hello-pybind11)
> `[](https://github.com/scikit-build/scikit-build-sample-projects/tree/master/projects/hello-pybind11)`

We really need to find a good tutorial about using _skbuild_ with _pybind11_ to build C++ extensions that use OpenCV that we can follow to build our extensions.
